### ISSUE TITLE
Question Re: Setting encoder properties after usage

### ISSUE THREAD

**Issue by idhland** [GPUOpen-LibrariesAndSDKs/AMF#301]
Hello! Are we supposed to be able to set encoder properties after setting "Usage" ( and before Init ). Different OS SKUs for Windows are giving my team different results. I was curious about any insight that could be provided. Thanks!

**Comment by rhutsAMD** [comment]
Hi! Yes, the different "Usages" are meant to provide a set of default values, after which you may further modify and set properties based on different categories. 

**Static Properties:**
- Should be set before encoder initialization

**Dynamic Properties:**
- Can be set at any time

**Per-Submission Properties:**
- Can be set on the AMFSurface being provided as input with SubmitInput()
- Applies only once

**Comment by idhland** [comment]
Hey Roman, thanks for getting back to me so quickly! I thought that was the intention of the API. We're running into issues with this on a cloud VM SKU so I wanted to check first before going back to them. Basically we're unable to set properties after choosing a "Usage". Thanks again.

**Comment by idhland** [comment]
That makes sense. Let me ping them first to see if this is on their radar or if there's a reason for it. I'll get back to you.

**Comment by idhland** [comment]
Hey Roman, I think we found our issue. It looks like AMF_VIDEO_ENCODER_RATE_CONTROL_METHOD (and possibly other rate control parameters) cannot be set if the usage mode is set to AMF_VIDEO_ENCODER_USAGE_ULTRA_LOW_LATENCY. This only happens on GCN cards though. RDNA seems to be fine. Could you provide more context on the this subject? I didn't see anything in the documentation about this but I could have missed it.

**Comment by bisher13** [comment]
I agree while i am using my rx580 via FFmpeg encoder if i used the ultra latency used the most settings ignored

**Comment by MikhailAMD** [comment]
Stale issue. Please reopen if needed.
